scripts/update-imports.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+// Function to read a file and return its contents
+async function readFile(filePath: string): Promise<string> {
+  return fs.readFile(filePath, 'utf-8');
+}
+
+// Function to write data to a file
+async function writeFile(filePath: string, data: string): Promise<void> {
+  await fs.writeFile(filePath, data, 'utf-8');
+}
+
+async function processApiFile(filePath: string) {
+    const content = await readFile(filePath);
+    const updatedContent = content.replace(/import \{ OpenApiRequestBuilder \} from '@sap-cloud-sdk\/openapi';/g, "import { OpenApiRequestBuilder } from '@sap-ai-sdk\/core';");
+  await writeFile(filePath, updatedContent);
+}
+
+
+// Function to recursively traverse the directory and apply transformations
+async function traverseDirectory(dirPath: string): Promise<void> {
+  const files = await fs.readdir(dirPath);
+
+  for (const file of files) {
+    const filePath = path.join(dirPath, file);
+
+    try {
+        const stat = await fs.stat(filePath);
+  
+        if (stat.isDirectory() && path.basename(filePath) !== 'schema') {
```

Comment:
Given the remaining purpose of the script, do you think it might be a little overkill to check if it's schema & index, or do you think the improved efficiency is worth the additional complexity? I don't really have a preference.